### PR TITLE
focus-visible

### PR DESCRIPTION
**Pull request by kimlarocca**
re: our convo on slack. set it up so we can use this in any component

### CODE REVIEW

File: src/components/VAccordion.vue [hunk 1]
```diff
@@ -35,6 +35,7 @@
 
 <script>
 import SimpleArrowDown from '../components/icons/SimpleArrowDown'
+import 'focus-visible'
```
Comment (walsh9) [inline src/components/VAccordion.vue:38]: I think we should import this at a higher level than this one specific icon, seems like it should be able to cover all focusable elements. At the very least it should be up in the accordion component where this icon actually becomes a focusable thing.

**Review by walsh9 — CHANGES_REQUESTED** [review]

File: src/components/VAccordion.vue (hunk 1 above)
Comment (kimlarocca) [reply src/components/VAccordion.vue:38]: then it would have to be imported on the client site not in the design system - right?

Comment (walsh9) [reply src/components/VAccordion.vue:38]: i think we can just move it to the accordion for now. But this icon on its own isn't inherently a focusable control, the focus is happening to an instance of this icon and that instance's role as part of the accordion component.

I'm assuming Vue is smart enough here that this isn't getting loaded multiple times if it shows up in multiple places...

Comment (walsh9) [reply src/components/VAccordion.vue:38]: ah nevermind i misread

**Review by walsh9 — APPROVED** [review]

**Pull request merged**